src/shared/ntdef.rs

```diff
@@ -176,8 +197,872 @@ pub const MAXLONGLONG: LONGLONG = 0x7fffffffffffffff;
+pub const SUBLANG_SPANISH_PUERTO_RICO: USHORT = 0x14;
+pub const SUBLANG_SPANISH_US: USHORT = 0x15;
+pub const SUBLANG_SWAHILI_KENYA: USHORT = 0x01;
+pub const SUBLANG_SWEDISH: USHORT = 0x01;
+pub const SUBLANG_SWEDISH_FINLAND: USHORT = 0x02;
+pub const SUBLANG_SYRIAC_SYRIA: USHORT = 0x01;
+pub const SUBLANG_TAJIK_TAJIKISTAN: USHORT = 0x01;
+pub const SUBLANG_TAMAZIGHT_ALGERIA_LATIN: USHORT = 0x02;
+pub const SUBLANG_TAMAZIGHT_MOROCCO_TIFINAGH: USHORT = 0x04;
+pub const SUBLANG_TAMIL_INDIA: USHORT = 0x01;
+pub const SUBLANG_TAMIL_SRI_LANKA: USHORT = 0x02;
+pub const SUBLANG_TATAR_RUSSIA: USHORT = 0x01;
+pub const SUBLANG_TELUGU_INDIA: USHORT = 0x01;
+pub const SUBLANG_THAI_THAILAND: USHORT = 0x01;
+pub const SUBLANG_TIBETAN_PRC: USHORT = 0x01;
+pub const SUBLANG_TIGRIGNA_ERITREA: USHORT = 0x02;
+pub const SUBLANG_TIGRINYA_ERITREA: USHORT = 0x02;
+pub const SUBLANG_TIGRINYA_ETHIOPIA: USHORT = 0x01;
+pub const SUBLANG_TSWANA_BOTSWANA: USHORT = 0x02;
+pub const SUBLANG_TSWANA_SOUTH_AFRICA: USHORT = 0x01;
+pub const SUBLANG_TURKISH_TURKEY: USHORT = 0x01;
+pub const SUBLANG_TURKMEN_TURKMENISTAN: USHORT = 0x01;
+pub const SUBLANG_UIGHUR_PRC: USHORT = 0x01;
+pub const SUBLANG_UKRAINIAN_UKRAINE: USHORT = 0x01;
+pub const SUBLANG_UPPER_SORBIAN_GERMANY: USHORT = 0x01;
+pub const SUBLANG_URDU_PAKISTAN: USHORT = 0x01;
+pub const SUBLANG_URDU_INDIA: USHORT = 0x02;
+pub const SUBLANG_UZBEK_LATIN: USHORT = 0x01;
+pub const SUBLANG_UZBEK_CYRILLIC: USHORT = 0x02;
+pub const SUBLANG_VALENCIAN_VALENCIA: USHORT = 0x02;
+pub const SUBLANG_VIETNAMESE_VIETNAM: USHORT = 0x01;
+pub const SUBLANG_WELSH_UNITED_KINGDOM: USHORT = 0x01;
+pub const SUBLANG_WOLOF_SENEGAL: USHORT = 0x01;
+pub const SUBLANG_XHOSA_SOUTH_AFRICA: USHORT = 0x01;
+pub const SUBLANG_YAKUT_RUSSIA: USHORT = 0x01;
+pub const SUBLANG_YI_PRC: USHORT = 0x01;
+pub const SUBLANG_YORUBA_NIGERIA: USHORT = 0x01;
+pub const SUBLANG_ZULU_SOUTH_AFRICA: USHORT = 0x01;
+pub const SORT_DEFAULT: USHORT = 0x0;
+pub const SORT_INVARIANT_MATH: USHORT = 0x1;
+pub const SORT_JAPANESE_XJIS: USHORT = 0x0;
+pub const SORT_JAPANESE_UNICODE: USHORT = 0x1;
+pub const SORT_JAPANESE_RADICALSTROKE: USHORT = 0x4;
+pub const SORT_CHINESE_BIG5: USHORT = 0x0;
+pub const SORT_CHINESE_PRCP: USHORT = 0x0;
+pub const SORT_CHINESE_UNICODE: USHORT = 0x1;
+pub const SORT_CHINESE_PRC: USHORT = 0x2;
+pub const SORT_CHINESE_BOPOMOFO: USHORT = 0x3;
+pub const SORT_CHINESE_RADICALSTROKE: USHORT = 0x4;
+pub const SORT_KOREAN_KSC: USHORT = 0x0;
+pub const SORT_KOREAN_UNICODE: USHORT = 0x1;
+pub const SORT_GERMAN_PHONE_BOOK: USHORT = 0x1;
+pub const SORT_HUNGARIAN_DEFAULT: USHORT = 0x0;
+pub const SORT_HUNGARIAN_TECHNICAL: USHORT = 0x1;
+pub const SORT_GEORGIAN_TRADITIONAL: USHORT = 0x0;
+pub const SORT_GEORGIAN_MODERN: USHORT = 0x1;
+macro_rules! MAKELANGID { ($p:expr, $s:expr) => ((($s as USHORT) << 10) | ($p as USHORT)) }
+#[inline]
+pub fn MAKELANGID(p: USHORT, s: USHORT) -> LANGID { (s << 10) | p }
```

Comment:
Should these fns all be `unsafe` as well? Additionally, I think they need block indentation for their arguments.